Mathlib/LinearAlgebra/Vandermonde.lean

```diff
@@ -154,6 +154,15 @@ theorem det_vandermonde_ne_zero_iff [IsDomain R] {n : ℕ} {v : Fin n → R} :
   simp only [det_vandermonde_eq_zero_iff, Ne.def, not_exists, not_and, Classical.not_not]
 #align matrix.det_vandermonde_ne_zero_iff Matrix.det_vandermonde_ne_zero_iff
 
+theorem det_vandermonde_eq_det_vandermonde_add {n : ℕ} (v : Fin n → R) (a : R) :
+    (Matrix.vandermonde v).det = (Matrix.vandermonde fun i ↦ v i + a).det := by
+  simp [Matrix.det_vandermonde]
```

Comment:
Please swap these statements around. Then they can become simp-lemmas. Also, you can then shorten the name to `det_vandermonde_add`.

Reply:
done